File: 01_ProgrammingBasics/_04_LoopsPartOne/_09_LeftAndRightSum.java
package _01_ProgrammingBasics._04_LoopsPartOne;

import java.util.Scanner;

public class _09_LeftAndRightSum {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        int n = Integer.parseInt(scan.nextLine());

        int leftSum = 0;
        int rightSum = 0;

        // Read and calculate the sum of the left sequence
        for (int i = 1; i <= n; i++) {
            int currentNum = Integer.parseInt(scan.nextLine());
            leftSum += currentNum;
        }

        // Read and calculate the sum of the right sequence
        for (int i = 1; i <= n; i++) {
            int currentNum = Integer.parseInt(scan.nextLine());
            rightSum += currentNum;
        }

        // Compare the sums and print the result
        if (leftSum == rightSum) {
            System.out.printf("Yes, sum = %d", rightSum);
        } else {
            System.out.printf("No, diff = %d", Math.abs(leftSum - rightSum));
        }

        scan.close();
    }
}

File: 23_C#Basics/_04_LoopsPartOne/_09_LeftAndRightSum/_09_LeftAndRightSum.cs
using System;

public class _09_LeftAndRightSum
{
    public static void Main(string[] args)
    {
        // Read input value
        int n = int.Parse(Console.ReadLine());

        int leftSum = 0;
        int rightSum = 0;

        // Read and calculate the sum of the left sequence
        for (int i = 1; i <= n; i++)
        {
            int currentNum = int.Parse(Console.ReadLine());
            leftSum += currentNum;
        }

        // Read and calculate the sum of the right sequence
        for (int i = 1; i <= n; i++)
        {
            int currentNum = int.Parse(Console.ReadLine());
            rightSum += currentNum;
        }

        // Compare the sums and print the result
        if (leftSum == rightSum)
        {
            Console.WriteLine($"Yes, sum = {rightSum}");
        }
        else
        {
            Console.WriteLine($"No, diff = {Math.Abs(leftSum - rightSum)}");
        }
    }
}
